install/kubernetes/cilium/files/nodeinit/prestop.bash

```diff
@@ -30,9 +30,18 @@ rm -f /tmp/node-init.cilium.io
 touch /tmp/node-deinit.cilium.io
 
 {{- if .Values.nodeinit.reconfigureKubelet }}
-echo "Changing kubelet configuration to --network-plugin=kubenet"
-sed -i "s:--network-plugin=cni\ --cni-bin-dir={{ .Values.cni.binPath }}:--network-plugin=kubenet:g" /etc/default/kubelet
-echo "Restarting kubelet..."
+if [[ command -v gke && command -v containerd ]];
```

Comment:
Bash's `[[` will complain here, and I think `then` cannot be omitted:

```suggestion
if command -v gke && command -v containerd; then
```

Question: shouldn't this test look like the one in `startup.bash`?

Reply:
That's correct — thank you for spotting it.